emulator_info symbols missing

0.244 built fine on NetBSD.

In the same environment, when building 0.245, I see the following linker error:
```
Linking mame...
ld: ../../../../netbsd/bin/x64/Release/libemu.a(debugcon.o): in function `debugger_console::debugger_console(running_machine&)':
debugcon.cpp:(.text+0x1e4e): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(debugcpu.o): in function `device_debug::instruction_hook(unsigned int)':
debugcpu.cpp:(.text+0xa55c): undefined reference to `emulator_info::periodic_check()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(empty.o): in function `empty_state::machine_start()':
empty.cpp:(.text._ZN11empty_state13machine_startEv[_ZN11empty_state13machine_startEv]+0x36): undefined reference to `emulator_info::display_ui_chooser(running_machine&)'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(ioport.o): in function `ioport_manager::record_init()':
ioport.cpp:(.text+0x71cf): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(rendlay.o): in function `layout_file::layout_file(device_t&, util::xml::data_node const&, char const*, char const*)':
rendlay.cpp:(.text+0xe358): undefined reference to `emulator_info::layout_script_cb(layout_file&, char const*)'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(screen.o): in function `screen_device::finalize_burnin()':
screen.cpp:(.text+0x6e62): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(sound.o): in function `sound_manager::update(int)':
sound.cpp:(.text+0x4601): undefined reference to `emulator_info::sound_hook()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(video.o): in function `video_manager::exit()':
video.cpp:(.text+0x32a): undefined reference to `emulator_info::standalone()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(video.o): in function `video_manager::save_snapshot(screen_device*, util::core_file&)':
video.cpp:(.text+0x442c): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(video.o): in function `video_manager::frame_update(bool)':
video.cpp:(.text+0x65b7): undefined reference to `emulator_info::draw_user_interface(running_machine&)'
ld: video.cpp:(.text+0x65bc): undefined reference to `emulator_info::frame_hook()'
ld: video.cpp:(.text+0x6612): undefined reference to `emulator_info::periodic_check()'
ld: video.cpp:(.text+0x6695): undefined reference to `emulator_info::periodic_check()'
ld: video.cpp:(.text+0x6794): undefined reference to `emulator_info::draw_user_interface(running_machine&)'
ld: video.cpp:(.text+0x6799): undefined reference to `emulator_info::frame_hook()'
ld: video.cpp:(.text+0x6822): undefined reference to `emulator_info::draw_user_interface(running_machine&)'
ld: video.cpp:(.text+0x6827): undefined reference to `emulator_info::frame_hook()'
ld: video.cpp:(.text+0x685a): undefined reference to `emulator_info::periodic_check()'
ld: video.cpp:(.text+0x68c9): undefined reference to `emulator_info::periodic_check()'
ld: ../../../../netbsd/bin/x64/Release/libemu.a(recording.o): in function `movie_recording::create(running_machine&, screen_device*, movie_recording::format, std::unique_ptr<emu_file, std::default_delete<emu_file> >&&, bitmap_rgb32&)':
recording.cpp:(.text+0xadc): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/mame_mame/libosd_sdl.a(sdlmain.o): in function `sdl_osd_interface::init(running_machine&)':
sdlmain.cpp:(.text+0x120a): undefined reference to `emulator_info::get_build_version()'
ld: ../../../../netbsd/bin/x64/Release/mame_mame/libosd_sdl.a(sdlmain.o): in function `main':
sdlmain.cpp:(.text.startup+0x121): undefined reference to `emulator_info::start_frontend(emu_options&, osd_interface&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >&)'
ld: ../../../../netbsd/bin/x64/Release/mame_mame/libosd_sdl.a(video.o): in function `sdl_osd_interface::extract_video_config()':
video.cpp:(.text+0x24a6): undefined reference to `emulator_info::standalone()'
ld: ../../../../netbsd/bin/x64/Release/mame_mame/libosd_sdl.a(osdwindow.o): in function `osd_window::osd_window(running_machine&, int, std::shared_ptr<osd_monitor_info>, osd_window_config const&)':
osdwindow.cpp:(.text+0x1acb): undefined reference to `emulator_info::get_bare_build_version()'
gmake[2]: *** [mame.make:269: ../../../../../mame] Error 1
gmake[1]: *** [Makefile:1177: mame] Error 2
gmake[1]: Leaving directory '/scratch/emulators/mame/work/mame-mame0245/build/projects/sdl/mame/gmake-netbsd'
gmake: *** [makefile:1436: netbsd_x64] Error 2
*** Error code 2
```
Is there some operating system support missing? Or a sourcefile not added in some cases? Please advise! Thanks.

--- a/scripts/genie.lua
+++ b/scripts/genie.lua
@@ -1264,6 +1264,10 @@ configuration { "linux-*" }
 		end
 
 
+configuration { "netbsd" }
+		flags {
+			"LinkSupportCircularDependencies",
+		}
 
 configuration { "osx*" }
 		links {

